### ISSUE TITLE
make test_prep showing functions "not listed in embed.fnc"

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#19831]
As of the commit below:
```
commit 7008caa915ad99e650acf2aea40612b5e48b7ba2
Author:     Karl Williamson <khw@cpan.org>
AuthorDate: Sat May 28 15:30:19 2022 -0600
Commit:     Karl Williamson <khw@cpan.org>
CommitDate: Sun Jun 5 09:17:19 2022 -0600

    Remove deprecated functions
    
    Most of these have been deprecated for a long time; and we've never
    bothered to follow through in removing them.  This commit does that.
```
... we're starting to get some output to STDERR during `make test_prep`:
```
no docs for sv_pvutf8
Function 'find_rundefsvoffset', documented in pad.c, not listed in embed.fnc at autodoc.pl line 1759.
Function 'pad_compname_type', documented in mathoms.c, not listed in embed.fnc at autodoc.pl line 1759.
Function 'save_list', documented in pod/perlguts.pod, not listed in embed.fnc at autodoc.pl line 1759.
```
@khwilliamson, can you take a look?

Thank you very much.
Jim Keenan

**Comment by jkeenan** [comment]
> As of the commit below:
> 
> ```
> commit 7008caa915ad99e650acf2aea40612b5e48b7ba2
> Author:     Karl Williamson <khw@cpan.org>
> AuthorDate: Sat May 28 15:30:19 2022 -0600
> Commit:     Karl Williamson <khw@cpan.org>
> CommitDate: Sun Jun 5 09:17:19 2022 -0600
> 
>     Remove deprecated functions
>     
>     Most of these have been deprecated for a long time; and we've never
>     bothered to follow through in removing them.  This commit does that.
> ```
> 
> ... we're starting to get some output to STDERR during `make test_prep`:
> 
> ```
> no docs for sv_pvutf8
> Function 'find_rundefsvoffset', documented in pad.c, not listed in embed.fnc at autodoc.pl line 1759.
> Function 'pad_compname_type', documented in mathoms.c, not listed in embed.fnc at autodoc.pl line 1759.
> Function 'save_list', documented in pod/perlguts.pod, not listed in embed.fnc at autodoc.pl line 1759.
> ```

Configuration here was `sh ./Configure -des -Dusedevel -Duseithreads -Dcc=gcc12` (on FreeBSD-12).

**Comment by khwilliamson** [comment]
Fixed by 789738ed9c41fd6d8c4bb2608e31c156cfe55fff and d7facc808f155e75c17b5897933d29cf5052c5fd

**Comment by jkeenan** [comment]
> Fixed by [789738e](https://github.com/Perl/perl5/commit/789738ed9c41fd6d8c4bb2608e31c156cfe55fff) and [d7facc8](https://github.com/Perl/perl5/commit/d7facc808f155e75c17b5897933d29cf5052c5fd)

Confirmed. Thanks.
